src/annotator/adder.js

```diff
@@ -193,6 +189,56 @@ export class Adder {
     return { top, left, arrowDirection };
   }
 
+  /**
+   * Find a Z index value that will cause the adder to appear on top of any
+   * content in the document when the adder is shown at (left, top).
+   *
+   * @param {number} left - Horizontal offset from left edge of viewport.
+   * @param {number} top - Vertical offset from top edge of viewport.
+   * @return {number} - greatest zIndex (default value of 1)
+   */
+  _findZindex(left, top) {
+    if (document.elementsFromPoint === undefined) {
+      // In case of not being able to use `document.elementsFromPoint`,
+      // default to the greatest <integer> allowed in Opera (2^15).
+      // The <integer> value is currently enforced in the specification,
+      // and Opera has one of the smaller ranges.
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/integer#Syntax
+      // For a less outdated info see:
+      // https://joeist.com/2012/06/what-is-the-highest-possible-z-index-value/
+      return 32768;
```

Comment:
I think it would be clearer to remove this comment and just note that the fallback of 2^15 is chosen as an arbitrary large value. The note on MDN applies to a version of Opera (12.1) that is 8 years old and pre-dates Hypothesis!